[2024-04-25T23:26:35.656+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-25T23:26:35.679+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pipeline.etl.load manual__2024-04-25T22:52:20.828871+00:00 [queued]>
[2024-04-25T23:26:35.810+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pipeline.etl.load manual__2024-04-25T22:52:20.828871+00:00 [queued]>
[2024-04-25T23:26:35.810+0000] {taskinstance.py:2303} INFO - Starting attempt 3 of 3
[2024-04-25T23:26:35.820+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): etl.load> on 2024-04-25 22:52:20.828871+00:00
[2024-04-25T23:26:35.830+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=4994) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-04-25T23:26:35.830+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'pipeline', 'etl.load', 'manual__2024-04-25T22:52:20.828871+00:00', '--job-id', '214', '--raw', '--subdir', 'DAGS_FOLDER/air_quality/air_quality_dag.py', '--cfg-path', '/tmp/tmp6w64n_w9']
[2024-04-25T23:26:35.831+0000] {standard_task_runner.py:63} INFO - Started process 4996 to run task
[2024-04-25T23:26:35.831+0000] {standard_task_runner.py:91} INFO - Job 214: Subtask etl.load
[2024-04-25T23:26:35.863+0000] {task_command.py:426} INFO - Running <TaskInstance: pipeline.etl.load manual__2024-04-25T22:52:20.828871+00:00 [running]> on host cf1d83d83b27
[2024-04-25T23:26:35.923+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='pipeline' AIRFLOW_CTX_TASK_ID='etl.load' AIRFLOW_CTX_EXECUTION_DATE='2024-04-25T22:52:20.828871+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-25T22:52:20.828871+00:00'
[2024-04-25T23:26:35.924+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-25T23:26:35.934+0000] {RedShift.py:56} INFO - Loading data into Redshift
[2024-04-25T23:26:38.202+0000] {RedShift.py:71} INFO - Data inserted successfully
[2024-04-25T23:26:38.203+0000] {RedShift.py:77} INFO - Connection closed
[2024-04-25T23:26:38.214+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-04-25T23:26:38.215+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-25T23:26:38.223+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=pipeline, task_id=etl.load, execution_date=20240425T225220, start_date=20240425T232635, end_date=20240425T232638
[2024-04-25T23:26:38.249+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-25T23:26:38.263+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-04-25T23:26:38.265+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
